test/llvm_ir_correct/contig.f90

```diff
@@ -0,0 +1,5 @@
+! RUN: %flang -c %s -fsyntax-only
+module contig
+  implicit none
+  integer, dimension(:), pointer, contiguous, save :: ptr
+end module
```

Comment:
This test case should probably be moved into `test/sema/` I guess? Consider naming it `contiguous.f90`.

Reply:
I don't think such a detailed division of test cases would give any additional value to the classic flang, also in the past I've been placing more test cases like this one in this same directory. And adding test case to this directory does not require modifying or adding any other files.

I've deliberately avoided naming this test case 'conriguous' to prevent name clash with one of the f2008 keywords.